docs/examples/tar_and_transfer.rst

```diff
@@ -1,6 +1,50 @@
 Tar and Transfer
 ----------------
 
+This example highlights the usage of the Tar and Transfer Gladier tools in a single flow.
+Start the workflow by overriding the Gladier Client with a list of the
+tools you want to use. In this case, we want to use the Tar, and Transfer tools.
 
+.. code-block:: python
 
-Tar and Tranfer files
\ No newline at end of file
+   @generate_flow_definition
+   class TarAndTransfer(GladierBaseClient):
+       gladier_tools = [
+       'gladier_tools.posix.Tar',
+       'gladier_tools.globus.Transfer',
+        ]
```

Comment:
Can you include the imports here? Actually, this might be better as a complete python script which is then included in the docs. Here's an example from another rst: https://github.com/globus-gladier/gladier/blob/main/docs/gladier_client.rst

Reply:
Oh, additionally, can you run `flake8` on the code? Gladier Tools wasn't enforcing any style guides, but since Gladier and Gladier Tools are packages I'd like to start doing that. https://flake8.pycqa.org/en/latest/

Reply:
So, have the whole script at the top and then explain whats going on in the script?

Reply:
I do like the idea of it being a whole and complete script. That way everything can be copy-pasted and run as-is without modifications or hunting down all the required snippets.